Xamarin.Forms.Platform.iOS/Renderers/LabelRenderer.cs

```diff
@@ -469,5 +485,46 @@ void UpdateMaxLines()
 #endif
 			}
 		}
+
+		void UpdatePadding()
+		{
+			if (Element.Padding.IsEmpty)
+				return;
+
+#if __MOBILE__
+			var formsLabel = Control as FormsLabel;
+			if (formsLabel == null)
+			{
+				Debug.WriteLine($"{nameof(LabelRenderer)}: On iOS, a Label created with no padding will ignore padding changes");
```

Comment:
In a distant corner of my mind, I remember @samhouts saying the team would like to avoid slowing down the debugger with logs.

Reply:
@adrianknight89 this isn't part of any critical performance path (like layout update or something like that), and most importantly, it's very useful for anybody to understand what's happening.